worker-test/src/main/docker/rabbitmq/certs/RMQ-CA-cert.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
+MIIDYzCCAkugAwIBAgIUaLI9/09sNCFINmbiwg/tUCKE6fYwDQYJKoZIhvcNAQEL
+BQAwQTELMAkGA1UEBhMCVUsxEzARBgNVBAgMClNvbWUtU3RhdGUxDTALBgNVBAoM
+BEpNQ0MxDjAMBgNVBAMMBTEyMzQ1MB4XDTI0MDIyOTA5MjQyNloXDTI0MDYwODA5
+MjQyNlowQTELMAkGA1UEBhMCVUsxEzARBgNVBAgMClNvbWUtU3RhdGUxDTALBgNV
+BAoMBEpNQ0MxDjAMBgNVBAMMBTEyMzQ1MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8A
+MIIBCgKCAQEAstLGXp+9nc+SiDvQlayRovVjgYS19rl5EtOkUjE9dYk+ull1XyTg
+cwg7h0kKWb5rbG9+SKIoYQFbvz8n8MTr0DvGIETFZimy8PSo5/fIv5AP8t3u0Sh3
+F4NvBWfLnkXA+P59i09SWOQcI9ivzvT+AmcMUjGuEEEhT9Ak47HfBcoliq/clMhh
+nvpFKmi3VWoqNRPL9OREFEFjLlaz1AqY1iy0gf8REBls604qrzkFJk6uoIbFY36r
+GGXsE4CWcTSlXnBv9Djkz6v3hsSlSS7FgYxSll84hBluFK1Eg5kV9teE+re24s9P
+YmeB5sXFC/SImEVsVRr3vEAg0nxQCCoYjQIDAQABo1MwUTAdBgNVHQ4EFgQUOMUp
+xYtx5AhhswWMTe7Tza7iWXEwHwYDVR0jBBgwFoAUOMUpxYtx5AhhswWMTe7Tza7i
+WXEwDwYDVR0TAQH/BAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAQQkORj6Ie/1i
+gTj5D7WBVWi0nr74UI3xNqN+UV8QnqVW+bITbmdr1/kIlzlHLUIyWvgVK9BWUmgo
+LFynQgv0LDpQSpbvt8/01YqdqRHxCSKxNTz0nX7ZHOM3hSM8htRdrS7g8eFOmpsO
+UCOfnq0ajbk5U2qu0FdMLerfwqVoT3wW0UtLrOfiGNKWuIweVPG1MqPUAVRqQ5Aj
+w8TbgjrTQiazahABCTApJoVc5PVJt59S3Iz/oI85il8XMCs7ohllj2VyWubk4G25
+TgBzeNfYr0xTM9bLYsVARe2y/WxF2Qt1+XfTIp42XCLKHGwqUvgcas9cjC9Tsh8A
+scXkgT9gPg==
+-----END CERTIFICATE-----
```

Comment:
Why are we checking in certificates?  Can't we generate these during the build?

Reply:
I was just trying this on first pass but spoke to Andy about spinning up a `keystore` image to generate certificates. Just hadn't checked it in until this morning.